Spigot-Server-Patches/0666-add-DragonEggFormEvent.patch

```diff
@@ -0,0 +1,44 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Trigary <trigary0@gmail.com>
+Date: Mon, 25 Jan 2021 14:53:57 +0100
+Subject: [PATCH] add DragonEggFormEvent
+
+
+diff --git a/src/main/java/net/minecraft/server/EnderDragonBattle.java b/src/main/java/net/minecraft/server/EnderDragonBattle.java
+index 7f11adbc5d8527b50f7657d46459516a471017b3..a50e01f5920809038205b5eba0704e88e4d145ca 100644
+--- a/src/main/java/net/minecraft/server/EnderDragonBattle.java
++++ b/src/main/java/net/minecraft/server/EnderDragonBattle.java
+@@ -15,6 +15,7 @@ import java.util.function.Predicate;
+ import javax.annotation.Nullable;
+ import org.apache.logging.log4j.LogManager;
+ import org.apache.logging.log4j.Logger;
++import io.papermc.paper.event.block.DragonEggFormEvent; // Paper - DragonEggFormEvent
+ 
+ public class EnderDragonBattle {
+ 
+@@ -358,9 +359,24 @@ public class EnderDragonBattle {
+             this.bossBattle.setVisible(false);
+             this.generateExitPortal(true);
+             this.n();
++            // Paper start - DragonEggFormEvent
++            BlockPosition eggPosition = this.world.getHighestBlockYAt(HeightMap.Type.MOTION_BLOCKING, WorldGenEndTrophy.a);
```

Comment:
obfhelper

Reply:
saw that too, but since its just copy pasted from below I didnt care

Reply:
Well I added a obfhelper